### ISSUE TITLE
virtualrealization.get_smry(time_index='last')

### ISSUE THREAD

**Issue by berland** [equinor/fmu-ensemble#135]
For a VirtualRealization computed from a delta ensemble with monthly summary, get_smry(time_index='last') gives ValueError: Requested frequency last not supported.
(from line 476 in get_smry_dates)

**Comment by berland** [comment]
This comes from the lines in virtualrealization.get_smry_dates():

`` 
489         # Parse every date to datetime, needed?
490         available_dates = [pd.to_datetime(x) for x in list(available_dates)]
491         start_date = min(available_dates)
492         end_date = max(available_dates)
493         pd_freq_mnenomics = {"monthly": "MS", "yearly": "YS", "daily": "D"}
494         if normalize:
495             raise NotImplementedError
496             # (start_date, end_date) = normalize_dates(start_date, end_date,
497             #                                         freq)
498         if freq not in pd_freq_mnenomics:
499             raise ValueError("Requested frequency %s not supported" % freq)
``` 
where support for `last` (and `first`) must be added, not only monthly, yearly and daily.

**Comment by asnyv** [comment]
closed through #133
